pkg/install/generate.go

```diff
@@ -237,6 +237,13 @@ func InstallerPodSpec(
 			},
 		)
 		env = append(env, oVirtCredsEnvVars(cd.Spec.Platform.Ovirt.CredentialsSecretRef.Name)...)
+		// TODO: design/implement a system to sync over ClusterImageSet base OS images into OpenStack
```

Comment:
This can already be done via `.spec.provisioning.installerEnv`. Do we need to special-case this?

Reply:
I haven't completely thought this through, but what I had in mind was something like having a clusterimageset defined will cause a pre-emptive upload of the RHCOS image to the OpenStack/oVirt/whatever environment, and ClusterDeployments would just use the pre-uploaded images automatically. It gets a little hazy as clusterdeployment A might be targeting oVirt environment A, and clusterdeployment B is targeting oVirt environment B. Hive would needs creds for all possible environments. Hive would need to know that a new clusterdeployment targets an environment that doesn't have a pre-uploaded image. I'm sure there are more things to think about...

But, it is great to know about this .spec.provisioning.installerEnv to simplify any future testing.

Reply:
I guess I don't understand why we are *adding* commented out code.

Reply:
I'll remove it now. I had put it in there as an easy-to-find marker for when I wanted to speed up my testing. But with this installerEnv field, it doesn't make sense. I'll remove it and open a Jira card for my longer-term dream of having this "just work".